scripts/dts/extract_dts_includes.py

```diff
@@ -294,6 +296,7 @@ def extract_single(node_address, prop, key, def_label):
             label = def_label + '_' + k
             if isinstance(p, str):
                 p = "\"" + p + "\""
+            add_compat_alias(node_address, k, label, prop_alias)
```

Comment:
Why do you think 'k' should be decorated?

Reply:
Unfortunately as best I can tell there isn't a dts that hits this code path, so not sure what its trying to do anymore.

Reply:
> Why do you think 'k' should be decorated?

Because otherwise the same alias would be added multiple times with different values, and effectively, after filtering, only the one for the last position from the list would get generated.

> Unfortunately as best I can tell there isn't a dts that hits this code path, so not sure what its trying to do anymore.

Yes, it is not currently used. And I have no clue what case this path was intended to handle. I only added the comment because this code was updated in this PR.